### DISCUSSION TITLE
about iTwin Snapshot.exe comond line

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3276] [Q&A]
now i want know how create Snapshot imodel from revit file by  command line

i try to open the C:\Users\Administrator\AppData\Local\imodelbridges\RevitBridge\iModelBridgeFwk.exe
  run command line  
what parameters and value i must input ???

![QQ截图20220228172732](https://user-images.githubusercontent.com/18022720/155958295-0a014f3a-0511-4952-b019-c8a136389ebf.png)

**Comment by jffmarker** [comment]
We don't normally encourage calling the Connectors directly... there are many arguments, and each synchronization will actually invoke a Connector several times for different purposes. Why are you interesting in running Connectors directly?

**Reply by Snailya** [reply]
Same problem.
We have a local network storage for different types of model. To let it able to render in a web app by directly selecting which the end user what to view, I need to implement a conversion service in backend to convert the file into bim model.

**Reply by jffmarker** [reply]
Our Product Manager recommends voting for https://itwinplatform.ideas.aha.io/ideas/ITWINGTM-I-139 to start with. We are also considering the effort involved to write and maintain enough documentation to support orchestrating the Bentley-authored Connectors.
